### PR TITLE
PSR-554 remove pension payments page

### PR DESCRIPTION
**Pull request by narendrapaduchuri**

### CODE REVIEW

File: app/controllers/nonsipp/memberpensionpayments/RemovePensionPaymentsController.scala [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+)(implicit ec: ExecutionContext)
+    extends PSRController
+    with I18nSupport {
+
+  private val form = RemovePensionPaymentsController.form(formProvider)
+
+  def onPageLoad(srn: Srn, index: Max300): Action[AnyContent] =
+    identifyAndRequireData(srn) { implicit request =>
+      val nameDOB = request.userAnswers.get(MemberDetailsPage(srn, index)).get
+      val totalContrib = request.userAnswers.get(TotalAmountPensionPaymentsPage(srn, index))
```
Comment (cyeniceri) [inline app/controllers/nonsipp/memberpensionpayments/RemovePensionPaymentsController.scala:58]: naming seems not correct here

File: app/navigation/nonsipp/PensionPaymentsReceivedNavigator.scala [hunk 2]
```diff
@@ -16,13 +16,15 @@
 
 package navigation.nonsipp
 
+import controllers.nonsipp.memberpensionpayments.RemovePensionPaymentsController
```
Comment (cyeniceri) [inline app/navigation/nonsipp/PensionPaymentsReceivedNavigator.scala:19]: this import is not used here

File: app/pages/nonsipp/memberpensionpayments/RemovePensionPaymentsPage.scala [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+package pages.nonsipp.memberpensionpayments
+
+import config.Refined.Max300
+import models.SchemeId.Srn
+import pages.Page
+
+case class RemovePensionPaymentsPage(srn: Srn, index: Max300) extends Page {
+
+  override def toString: String = "removePensionPayments"
```
Comment (cyeniceri) [inline app/pages/nonsipp/memberpensionpayments/RemovePensionPaymentsPage.scala:25]: do we need this ? Are we using "removePensionPayments" anywhere such as in our userAnswer ?

**Review by cyeniceri — CHANGES_REQUESTED** [review]

File: app/pages/nonsipp/memberpensionpayments/RemovePensionPaymentsPage.scala (hunk 3 above)
Comment (narendrapaduchuri) [reply app/pages/nonsipp/memberpensionpayments/RemovePensionPaymentsPage.scala:25]: It seems we don't need this, just checking now after that will push the code. Thanks

**Review by cyeniceri — APPROVED** [review]

**Pull request merged**